.env

```
@@ -38,8 +38,8 @@ REACT_APP_DOMAIN_REGEX_LOCAL="^(:?localhost:\d{2,5}|(?:127|192)(?:\.[0-9]{1,3}){
 REACT_APP_DOMAIN_REGEX_PR="^pr\d+--cowswap\.review"
 REACT_APP_DOMAIN_REGEX_DEV="^cowswap\.dev"
 REACT_APP_DOMAIN_REGEX_STAGING="^cowswap\.staging"
-REACT_APP_DOMAIN_REGEX_PROD="^cowswap\.exchange$"
-REACT_APP_DOMAIN_REGEX_BARN="^barn\.cowswap\.exchange$"
+REACT_APP_DOMAIN_REGEX_PROD="(^cowswap\.exchange$)|(^swap\.cow\.fi$)"
```

ubernit: the $ can be extracted out of the OR

```suggestion
REACT_APP_DOMAIN_REGEX_PROD="(^cowswap\.exchange)|(^swap\.cow\.fi)$"
```

---

Why not also pull out the `^`?:
```suggestion
REACT_APP_DOMAIN_REGEX_PROD="^(cowswap\.exchange|swap\.cow\.fi)$"
```

I do think, in this case, you need to group around everything that is `|`-ed.